src/sonic-config-engine/sonic-cfggen

```diff
@@ -304,7 +304,10 @@ def main():
             }}}
         # The ID needs to be passed to the SAI to identify the asic.
         if asic_name is not None:
-            hardware_data['DEVICE_METADATA']['localhost'].update(asic_id=asic_id)
+            pci_id = get_npu_pci_device_id(asic_id)
+            # if the pci_id obtained is None, use the asic_id itself.
+            device_id = asic_id if pci_id is None else pci_id
```

Comment:
When we dont get  the pic_id do we want to throw an error instead of using asic_id. 
Can SAI understand both pci_id and asic_id ?

Reply:
> We need to make any changes for multi ASIC VS to work ?

Yes, what I find is we will need some work to be done with the multi-asic VS to pass the instance ID to orchagent. Additionaly to use pci-id as the instance id to pass to the sai stub in VS, we need to update the asic.conf file for VS platform. 
We could take this as a separate activity ?